.changelog/4607.bugfix.md

```diff
@@ -0,0 +1,5 @@
+governance: Allow unregistered nodes in votes
```

Comment:
Should probably be marked as breaking.

Reply:
Even when ci passes, this changelog snippet nonsense terrorizes me.